src/main/kotlin/org/wycliffeassociates/otter/common/domain/content/WorkbookFileNamerBuilder.kt

```diff
@@ -0,0 +1,97 @@
+package org.wycliffeassociates.otter.common.domain.content
+
+import io.reactivex.schedulers.Schedulers
+import org.wycliffeassociates.otter.common.data.workbook.*
+
+class WorkbookFileNamerBuilder: FileNamer.Builder() {
+
+    private var buildLevel: BuildLevel = BuildLevel.EMPTY
```

Comment:
We talked about holding state in this object, and there may be a better way.

Reply:
Yes, we are already essentially already holding state in the WorkbookViewModel, so we can just set everything all at once. We all agree on this now! :)